lesson_05/nilejackson/hw5.md

```diff
@@ -0,0 +1,31 @@
+## USER Stories
+
+## User Story examples
+
+* Being a self-proclaimed sneaker enthusiast means participating in sneaker raffles which means purchasing sneakerbots to give myself the best chance of 'landing a new pair'. I want a sneakerbot that comes with clear concise directions in the manual, recommended best practices for running proxies in that paricular bot and an encrypted activation key.
+
+## Acceptance Criteria: 
+
+* - a detailed yet concise usermanual, that is easy to read and informative. 
```

Comment:
The formatting is inconsistent throughout and is missing proper capitalization. Please fix.